libs/langchain/langchain/retrievers/document_compressors/chain_extract.py

```diff
@@ -64,7 +64,9 @@ def compress_documents(
         compressed_docs = []
         for doc in documents:
             _input = self.get_input(query, doc)
-            output = self.llm_chain.predict_and_parse(**_input, callbacks=callbacks)
+            output = self.llm_chain.predict(**_input, callbacks=callbacks)
```

Comment:
you should be able to replace predict and parse calls with just `invoke(_input, config={"callbacks": callbacks})`